### PR TITLE
Adds Blockchain source and two BTC commands

### PR DESCRIPTION
**Pull request by jose-donato**
# Description
Gamestonk Terminal still lacks in terms of info related to bitcoin blockchain. This PR adds [Blockchain](https://blockchain.info/api) source and two BTC commands to `onchain` menu:
* `btccp`: displays the circulating supply of bitcoin
image

* `btcct`: displays the transactions per day
image

# Checklist:

- [x] Update [our Hugo documentation](https://gamestonkterminal.github.io/GamestonkTerminal/) following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/scripts).

# How has this been tested?
Running the commands in the terminal with various configurations.

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

**Comment by jose-donato** [comment]
PR created 20min ago and is already out-of-date with main 🤣

File: gamestonk_terminal/cryptocurrency/onchain/blockchain_view.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.config_plot import PLOT_DPI
+
+
+def display_btc_circulating_supply(since: int, until: int, export: str) -> None:
+    """Returns BTC circulating supply [Source: https://api.blockchain.info/]
+
+    Parameters
+    ----------
+    export : str
```
Comment (DidierRLopes) [inline gamestonk_terminal/cryptocurrency/onchain/blockchain_view.py:23]: Some parameters missing here

**Comment by jose-donato** [comment]
@JakubPluta do you know this source? has some great data. added these two to start, can probably add more: miners revenue, hashrate distribution, Market Value to Realised Value, etc

File: gamestonk_terminal/cryptocurrency/onchain/blockchain_view.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        "btccp",
+        df,
+    )
+
+
+def display_btc_confirmed_transactions(since: int, until: int, export: str) -> None:
+    """Returns BTC confirmed transactions [Source: https://api.blockchain.info/]
+
+    Parameters
+    ----------
```
Comment (DidierRLopes) [inline gamestonk_terminal/cryptocurrency/onchain/blockchain_view.py:65]: parameters missing here

**Review by DidierRLopes — APPROVED** [review]
Minor stuff, apart from that looks good!

**Pull request merged**